packages/cojson/src/coValues/coMap.ts

```diff
@@ -268,34 +289,21 @@ export class RawCoMapView<
   }
 
   /** @category 5. Edit history */
-  nthEditAt<K extends keyof Shape & string>(
-    key: K,
-    n: number,
-  ):
-    | {
-        by: RawAccountID | AgentID;
-        tx: TransactionID;
-        at: Date;
-        value?: Shape[K];
-      }
-    | undefined {
-    const ops = this.timeFilteredOps(key);
-    if (!ops || ops.length <= n) {
+  nthEditAt<K extends keyof Shape & string>(key: K, n: number) {
+    const ops = this.getOps()[key];
```

Comment:
Switched to use `this.getOps` to skip the filtering operation.

We can check the "atTimeFilter" here